backends/dnf/pk-backend-dnf.c

```diff
@@ -203,19 +205,22 @@ remove_old_cache_directories (PkBackend *backend, const gchar *release_ver_str)
 		return;
 	}
 
-	/* parse the version, while being careful to not trip over for any
-	 * non-numeric strings that we don't know how to handle, e.g. "7.5" */
-	if (!g_ascii_string_to_unsigned (release_ver_str, 10, 1, 1000, &release_ver, &error)) {
-		g_debug ("failed to parse current release version: %s", error->message);
+	/* open directory */
+	cache_dir = g_dir_open ("/var/cache/PackageKit", 0, &error);
```

Comment:
Then it LGTM.

Reply:
Ahh, I recall why it was like that: it's so that if we install packagekit with jhbuild to a different prefix it would still use the same cache dir as the system installation.